pkgs/top-level/all-packages.nix

```diff
@@ -159,6 +159,8 @@ in
 
   demoit = callPackage ../servers/demoit { };
 
+  deviceTree = callPackage ../os-specific/linux/device-tree.nix {};
```

Comment:
I don't know what's the better implementation for the "API" in `all-packages.nix` for providing device trees.

I'm thinking that yes, helpers like `applyOverlays` is fine under `deviceTree`, but thinking that the raspberry pi blobs (and possibly others in the future) shouldn't be found under that name. Just like we don't have `linux.v4_4` but `linux_4_4`. `raspberryPiDtbs` and `raspberryPiOverlays` might be better names, but suggestions welcome.

Reply:
I've moved these packages, does the new organization look better?